src/utils/mapper_functions.ts

```diff
@@ -25,5 +25,5 @@ export function folderToNameAndIdMap(entity: ArFSFileOrFolderEntity): FolderName
 }
 
 export function fileToNameAndIdMap(entity: ArFSFileOrFolderEntity): FileNameAndId {
-	return { fileId: entity.entityId, fileName: entity.name };
+	return { fileId: entity.entityId as FileID, fileName: entity.name };
```

Comment:
Is this casting fine?

Reply:
Appears to be unnecessary any more. I can excise all examples of this.

Reply:
91faa5f